### ISSUE TITLE
Cannot remove toolbar items

### ISSUE THREAD

**Issue by AryaTwirl** [losnoco/Cog#327]
Cannot remove toolbar items other than the flexible space or space

1. Right click on toolbar and choose customise toolbar
2. Drag one of the toolbar icons away from toolbar to remove it
3. It cannot be removed. The poof animation never appears, and the icon remains in place.

**Expected behavior**
The icon should be removed

**Version information:***
 - macOS version: 12.5.1
 - Cog version: Newest version until somewhere late July

**Comment by AryaTwirl** [comment]
If that's the case, I'll close the issue, then. Thank you very much for the detailed explanation.

**Comment by kode54** [comment]
I can try redoing my builds with 13.3.1, but then I'll need to do them both with the CI, because I can't run older than 14, because I'm running the Ventura beta on my Mac.

**Comment by kode54** [comment]
I have rebuilt both the web and the App Store versions with 13.3.1, please test.

**Comment by AryaTwirl** [comment]
Alright, will test it tonight. Thank you so much for the effort :)
________________________________

**Comment by AryaTwirl** [comment]
I'll close the issue as completed. The customize toolbar function works as intended with September 7th update.

**Comment by kode54** [comment]
Not sure if I need to reopen this, but Xcode 14.0.1 seems to build binaries capable of having their toolbar reconfigured? I will reset the Xcode Cloud to Latest Release and see how that goes for future versions.

**Comment by andrienko** [comment]
Weirdly enough, I can remove some items, and cannot remove some other items.
Is there an option to do this manually? Is there some sort of config file or something?

**Comment by kode54** [comment]
Nope. And this is a macOS bug. And it doesn't affect macOS Ventura.

**Comment by kode54** [comment]
Maybe someone else can figure this out?
